packages/@react-aria/virtualizer/src/ScrollView.tsx

```diff
@@ -26,6 +26,7 @@ import React, {
 import {Rect, Size} from '@react-stately/virtualizer';
 import {useLayoutEffect} from '@react-aria/utils';
 import {useLocale} from '@react-aria/i18n';
+import {useResizeObserver} from '@react-spectrum/utils';
```

Comment:
Probably shouldn't import from react-spectrum in react-aria... do we need to move this util?